### ISSUE TITLE
Custom dev client crashes on Android 8 device

### ISSUE THREAD

**Issue by wnikola** [expo/expo#13965]
### Summary

When I try to open the custom dev client I installed on my Android 8 Samsung Galaxy S7 Edge it immediately crashes and I get an error saying "[app-name] has stopped". I built the client using eas.

### Managed or bare workflow? If you have made manual changes inside of the `ios/` or `android/` directories in your project, the answer is bare!

managed

### What platform(s) does this occur on?

Android

### Package versions

```
    "@react-native-community/masked-view": "0.1.10",
    "@react-navigation/bottom-tabs": "^5.11.10",
    "@react-navigation/native": "^5.9.4",
    "@react-navigation/stack": "^5.14.5",
    "axios": "^0.21.1",
    "expo": "^42.0.0",
    "expo-app-loading": "^1.0.3",
    "expo-av": "~9.2.3",
    "expo-dev-client": "^0.4.7",
    "expo-font": "~9.2.1",
    "expo-image-picker": "~10.2.2",
    "expo-linking": "~2.3.1",
    "expo-secure-store": "~10.2.0",
    "expo-status-bar": "~1.0.4",
    "expo-web-browser": "~9.2.0",
    "lottie-react-native": "4.0.2",
    "mobx": "^6.3.2",
    "mobx-react-lite": "^3.2.0",
    "native-base": "^3.0.2",
    "react": "16.13.1",
    "react-dom": "16.13.1",
    "react-native": "https://github.com/expo/react-native/archive/sdk-42.0.0.tar.gz",
    "react-native-gesture-handler": "~1.10.2",
    "react-native-reanimated": "~2.2.0",
    "react-native-safe-area-context": "3.2.0",
    "react-native-screens": "~3.4.0",
    "react-native-svg": "12.1.1",
    "react-native-web": "~0.13.12",
    "styled-components": "^5.3.0",
    "styled-system": "^5.1.5"
```

### Environment

```
    Expo CLI 4.9.1 environment info:
    System:
      OS: Linux 5.10 Manjaro Linux
      Shell: 5.8 - /bin/zsh
    Binaries:
      Node: 16.5.0 - /usr/bin/node
      Yarn: 1.22.11 - /usr/bin/yarn
      npm: 7.20.2 - /usr/bin/npm
    IDEs:
      Android Studio: 4.0 AI-193.6911.18.40.6514223
    npmPackages:
      expo: ^42.0.0 => 42.0.3 
      react: 16.13.1 => 16.13.1 
      react-dom: 16.13.1 => 16.13.1 
      react-native: https://github.com/expo/react-native/archive/sdk-42.0.0.tar.gz => 0.63.2 
      react-native-web: ~0.13.12 => 0.13.18 
    npmGlobalPackages:
      expo-cli: 4.9.1
    Expo Workflow: managed
```

### Reproducible demo or steps to reproduce from a blank project

1. Create a new project
2. Build a dev client using eas
3. Install the dev client on an Android 8 device
4. Try to open the client

### Stacktrace (if a crash is involved)

No stacktrace

**Comment by tcdavis** [comment]
Thanks for the report!
I can reproduce.  The root issue is 

08-11 14:38:34.980  8336  8336 E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.thetc.and8/expo.modules.devmenu.DevMenuActivity}: java.lang.IllegalStateException: Only fullscreen opaque activities can request orientation

**Comment by wnikola** [comment]
Hey thanks for the fix! When can we expect it to go live?

**Comment by lukmccall** [comment]
Hey @wnikola, I think we will publish a new version after finishing adding support for the React Native 0.65. It should be somewhere in the next week 😉

**Comment by wnikola** [comment]
Great, thanks! :slightly_smiling_face:

**Comment by wnikola** [comment]
@tcdavis Hey can you let me know how you got that stacktrace? I applied lukmccall's patch locally and I can open the client now, but it crashes when I try to run the project and I'd like to investigate further.

**Comment by esamelson** [comment]
Hey @wnikola -- apologies for the delay, this fix is now live in expo-dev-client@0.5.1. Let us know if you run into any further issues!
